app/scripts/presenters/visit_presenter.js

```diff
@@ -3,12 +3,20 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
-  'underscore'
-], function (_) {
+  'underscore',
+  'lib/image_proxy'
+], function (_, imageProxy) {
   'use strict';
 
   var MINIMUM_IMAGE_ENTROPY = 1.75;
-  var MINIMUM_IMAGE_WIDTH = 270;
+  var IMAGE_DISPLAY_WIDTH = 270;
+  var IMAGE_DISPLAY_HEIGHT = 180;
```

Comment:
160px is the width, and it will calculate the height in that function. The largest we ever use is 270px wide and in this case we let the browser scale the images down to 160px.